page.php

```diff
@@ -76,7 +76,8 @@
 
 // P4 Campaign/dataLayer fields.
 $context['cf_campaign_name'] = $page_meta_data['p4_campaign_name'][0] ?? '';
-$context['cf_project_id']    = $page_meta_data['p4_global_project_tracking_id'][0] ?? '';
+$context['cf_project_id']    = $page_meta_data['p4_global_project_tracking_id'][0] ?? 'not set';
+$context['cf_local_project'] = $page_meta_data['p4_local_project'][0] ?? '';
```

Comment:
Shouldn't this also fall back to `'not set'`?

Reply:
by default if no smartsheet ID configured, the default value would be set as 'not set' from "-Select Local Project-" dropdown option, but we will also add one more level of fallback at here.